cmd/node/metrics/machineStatistics.go

```diff
@@ -8,10 +8,11 @@ import (
 	"github.com/ElrondNetwork/elrond-go/core/appStatusPolling"
 	"github.com/ElrondNetwork/elrond-go/core/check"
 	"github.com/ElrondNetwork/elrond-go/core/statistics/machine"
+	"github.com/ElrondNetwork/elrond-go/sharding"
 )
 
-// StartMachineStatisticsPolling will start read information about current  running machini
-func StartMachineStatisticsPolling(ash core.AppStatusHandler, pollingInterval time.Duration) error {
+// StartMachineStatisticsPolling will start read information about current  running machine
```

Comment:
remove a blank space between `current` and `running`

Reply:
done.